tests/tensorflow2/test_keras.py

```diff
@@ -396,7 +397,8 @@ def test_gradtape_persistent(out_dir, saveall):
 @pytest.mark.slow
 @pytest.mark.parametrize("saveall", [True, False])
 def test_keras_fit(out_dir, tf_eager_mode, saveall):
-    hook = smd.KerasHook(out_dir=out_dir, save_all=saveall)
+    save_config = SaveConfig(save_interval=1) if saveall else None
```

Comment:
might be overkill for CI?

Reply:
We run only 10 steps 

> > > Please fix the build failure.
> > 
> > 
> > The build failure is due to the fact that the pipelines currently do not use the custom AWS TF version
> 
> so the tests would fail on vanilla TF too?

Yes, this feature will only be supported on AWS TF

Reply:
so the vanilla container codebuild will fail. can you add checks to ensure that it works for both versions of TF ?